test/testPublishCli.js

```diff
@@ -214,4 +214,21 @@ describe('hlx publish', () => {
 
     sinon.assert.calledWith(mockPublish.withDispatchVersion, 'ci1');
   });
+
+  it('hlx publish fails with empty serviceid string of any length', (done) => {
+    new CLI()
+      .withCommandExecutor('publish', mockPublish)
+      .onFail((err) => {
+        assert.ok(!err || err.indexOf('required'));
+        done();
+      })
+      .run(['publish',
+        '--wsk-auth', 'secret-key',
+        '--wsk-namespace', 'hlx',
+        '--fastly-auth', 'secret-key',
+        '--dispatch-version', 'ci1',
+        '--fastly-serviceid', '',
+      ]);
+    sinon.assert.fail('publish with serviceid filled with empty spaces should have failed :(');
```

Comment:
What about the non-empty spaces? ;) 

Why not just `"publish with empty serviceid should fail."`?